src/main/scala/com/fortysevendeg/android/sqlite/SQLDroidBlob.scala

```diff
@@ -0,0 +1,60 @@
+package com.fortysevendeg.android.sqlite
+
+import java.io.{ByteArrayInputStream, InputStream, OutputStream}
+import java.sql.{SQLException, Blob, SQLFeatureNotSupportedException}
+
+class SQLDroidBlob(byteArray: Array[Byte]) extends Blob {
+
+  private[this] val arrayLength = Option(byteArray) map (_.length) getOrElse 0
+
+  def getBytes(pos: Long, length: Int): Array[Byte] = {
+    val newPos = pos - 1
```

Comment:
I don't like it, but is the documentation says :cry: [Blob.getBytes](http://docs.oracle.com/javase/7/docs/api/java/sql/Blob.html#getBytes(long,%20int))

I think that I miss the override prefix, sorry

On the other hand, a default value is allowed when override Java parameters?